Dockerfile

```diff
@@ -98,10 +98,11 @@ RUN npm install elm@0.18.0 \
 
 # Install PHP 7.4 and Composer
 ENV COMPOSER_ALLOW_SUPERUSER=1
+COPY script/install-composer /opt/composer/install-composer
```

Comment:
@feelepxyz 

What do you think about this instead?

```diff
diff --git a/Dockerfile b/Dockerfile
index 3222495c2..b8556517d 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -98,12 +98,10 @@ RUN npm install elm@0.18.0 \

 # Install PHP 7.4 and Composer
 ENV COMPOSER_ALLOW_SUPERUSER=1
-COPY script/install-composer /opt/composer/install-composer
+COPY --from=composer:1.10.9 /usr/bin/composer /usr/local/bin/composer
 RUN add-apt-repository ppa:ondrej/php \
     && apt-get update \
     && apt-get install -y php7.4 php7.4-cli php7.4-xml php7.4-json php7.4-zip php7.4-mbstring php7.4-intl php7.4-common php7.4-gettext php7.4-curl php7.4-bcmath php7.4-gmp php7.4-imagick php7.4-gd php7.4-redis php7.4-soap php7.4-ldap php7.4-memcached php7.4-sqlite3 php7.4-apcu php7.4-tidy php7.4-mongodb php7.4-zmq php7.4-mysql php7.4-imap php7.4-geoip \
-    && /opt/composer/install-composer \
-    && mv composer.phar /usr/local/bin/composer


 ### GO
diff --git a/script/install-composer b/script/install-composer
deleted file mode 100755
index bae15246e..000000000
--- a/script/install-composer
+++ /dev/null
@@ -1,17 +0,0 @@
-#!/bin/sh
-
-EXPECTED_CHECKSUM="e5325b19b381bfd88ce90a5ddb7823406b2a38cff6bb704b0acc289a09c8128d4a8ce2bbafcd1fcbdc38666422fe2806"
-php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
-ACTUAL_CHECKSUM="$(php -r "echo hash_file('sha384', 'composer-setup.php');")"
-
-if [ "$EXPECTED_CHECKSUM" != "$ACTUAL_CHECKSUM" ]
-then
-    >&2 echo 'ERROR: Invalid installer checksum'
-    rm composer-setup.php
-    exit 1
-fi
-
-php composer-setup.php --quiet
-RESULT=$?
-rm composer-setup.php
-exit $RESULT
```

Reply:
@localheinz yes ace! That's a lot neater.

Reply:
Glad you like it, @feelepxyz!